cabal-install/src/Distribution/Client/Init/Format.hs

```diff
@@ -244,6 +250,8 @@ mkPkgDescription opts pkgDesc =
       , "and can be different from the cabal-install (the tool) version and the"
       , "Cabal (the library) version you are using. As such, the Cabal (the library)"
       , "version used must be equal or greater than the version stated in this field."
+      , "Starting from the specification version 2.2, the cabal-version field must be"
+      , "the first thing in the cabal file."
       ]
```

Comment:
I took the liberty to add this extra comment here, stating that the `cabal-version` field should be the first line in the file.